model/project_ref.go

```diff
@@ -1788,6 +1788,14 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 			ProjectRefFilesIgnoredFromCacheKey:   p.FilesIgnoredFromCache,
 		}
 		if !isRepo && !p.UseRepoSettings() {
+			// Don't validate owner if user is defaulting page to repo
+			if p.Owner != "" {
+				allowedOrgs := evergreen.GetEnvironment().Settings().GithubOrgs
+				if err := validateOwner(p.Owner, allowedOrgs); err != nil {
```

Comment:
FYI https://github.com/evergreen-ci/evergreen/blob/d0385e39aff6c24036dc2d7d203e48bf4361eb0c/model/project_ref.go#L2107 already exists. Possibly we could drop this "no owner/repo specified" logic (or only run that line if the project is enabled).

Reply:
Yes, I updated that function to call `validateOwner` in order to split out the logic that checks if owner and repo are empty strings. This approach more cleanly handled the "default to repo" case where owner and repo are empty strings, which is independent of enabled status.

Reply:
Oops sorry my speed review was no good.